Assets/HoloToolkit/Utilities/Scripts/DirectionIndicator.cs

```diff
@@ -54,6 +54,7 @@ public void Awake()
 
             if (DirectionIndicatorObject == null)
             {
+                Debug.LogError("Direction Indicator failed to instantiate.");
```

Comment:
The line before instantiates  DirectionIndicatorObject = InstantiateDirectionIndicator(DirectionIndicatorObject); 
It's instantiating and setting the instantiated object back to the same variable name.  
We could set it to a different variable name.